framework/helpers/BaseStringHelper.php

```diff
@@ -497,4 +497,34 @@ public static function mb_ucwords($string, $encoding = 'UTF-8')
 
         return implode('', $parts);
     }
+
+    /**
+     * Masks a portion of a string with a repeated character.
+     * This method is multibyte-safe.
+     *
+     * @param string $string The input string.
+     * @param int $start The starting position from where to begin masking.
+     *                   This can be a positive or negative integer.
+     *                   Positive values count from the beginning,
+     *                   negative values count from the end of the string.
+     * @param int $offset The length of the section to be masked.
```

Comment:
`offset` is not a good name, suggesting the position, `length` would be better.

Reply:
Done